src/attributes/become-visible-custom-attribute.ts

```diff
@@ -25,38 +26,14 @@ export class BecomeVisibleCustomAttribute {
                 this._element.dispatchEvent(ev);
             }
         }
-    }
-
-    /**
-     * @see https://stackoverflow.com/questions/123999/how-to-tell-if-a-dom-element-is-visible-in-the-current-viewport
-     */
-    static elementInViewport(el) {
```

Comment:
FWIW: I have not yet seen it to be incorrect, but it _has_ featured prominently in my profiling sessions.
Good riddance, I say!